src/Reader/Reader.php

```diff
@@ -189,17 +197,14 @@ public static function useHttpConditionalGet($bool = true)
      */
     public static function import($uri, $etag = null, $lastModified = null)
     {
-        $cache       = self::getCache();
-        $client      = self::getHttpClient();
-        $client->resetParameters();
-        $headers = new ZendHttp\Headers();
-        $client->setHeaders($headers);
-        $client->setUri($uri);
+        $cache   = self::getCache();
```

Comment:
@harikt Additionally, you can continue using a solution like the Guzzle cache-subscriber with zend-feed; just don't inject a zend-cache adapter into the reader (it doesn't lazy-load one by default). The solution currently in `Reader` allows either approach, giving you an option for when the client doesn't have caching solutions built-in or easily added via plugins.